backend/compact-connect/stacks/persistent_stack/bulk_uploads_bucket.py

```diff
@@ -141,31 +144,43 @@ def _add_v1_ingest_object_events(self, event_bus: EventBus):
         """
         Read any objects that get uploaded and trigger ingest events
         """
-        stack: Stack = Stack.of(self)
+        stack: ps.PersistentStack = ps.PersistentStack.of(self)
         parse_objects_handler = PythonFunction(
             self, 'V1ParseObjectsHandler',
             description='Parse s3 objects handler',
             entry=os.path.join('lambdas', 'provider-data-v1'),
             index=os.path.join('handlers', 'bulk_upload.py'),
             handler='parse_bulk_upload_file',
             timeout=Duration.minutes(15),
+            alarm_topic=stack.alarm_topic,
             memory_size=1024,
             environment={
                 'EVENT_BUS_NAME': event_bus.event_bus_name,
                 **stack.common_env_vars
-            },
-            alarm_topic=stack.alarm_topic
+            }
         )
         self.grant_delete(parse_objects_handler)
         self.grant_read(parse_objects_handler)
         event_bus.grant_put_events_to(parse_objects_handler)
         self.log_groups.append(parse_objects_handler.log_group)
 
+        # We should specifically set an alarm for any failures of this handler, since it could otherwise go unnoticed.
```

Comment:
Yeah, I was thinking about the ingest process while I generated data for it and realized that there was a hole in our monitoring/alerting coverage. I figured, since it was a trivial change, I'd just address that while I was thinking about it.